core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -422,7 +422,7 @@ private[spark] object UIUtils extends Logging {
    * the whole string will rendered as a simple escaped text.
    *
    * Note: In terms of security, only anchor tags with root relative links are supported. So any
-   * attempts to embed links outside Spark UI, or other tags like &lt;script&gt; will cause in
+   * attempts to embed links outside Spark UI, or other tags like {@code <script>} will cause in
```

Comment:
- Java
  ![2016-11-28 2 24 00](https://cloud.githubusercontent.com/assets/6477701/20657065/55e66874-b576-11e6-9c8e-640e73fa6a8c.png)

Scaladoc not found